### PR TITLE
Initialise db and rabbit at app create time

### PR DESCRIPTION
**Pull request by jcox-dev**
# Motivation and Context

Setting up the database and rabbit is required as part of the CI app creation so this has been moved inside the `create_app` function. This also logs out which config object the setup has used and added a rabbit container to the compose file.

NB: found that the tests weren't patching the rabbit service correctly (resulting in false positives) so have updated accordingly.

# Test
`make test`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,6 @@ services:
 language: python
 python:
   - "3.6"
-before_install:
-  - docker-compose --verbose build
-  - docker-compose up -d
```
Comment (jcox-dev) [inline .travis.yml]: Now part of the `tox` script.

**Comment by jcox-dev** [comment]
[Log out config used in Python services](https://trello.com/c/Tctoorqs/251-log-out-config-used-in-python-services)

**Comment by jcox-dev** [comment]
[Log out the config used at CI startup](https://trello.com/c/232KnUQN/252-log-out-the-config-used-at-ci-startup)

**Comment by emilioward** [comment]
Whats the best way to tests this?

**Comment by jcox-dev** [comment]
@emilioward `make test` 👍

**Pull request merged**